bin/git-summary

```diff
@@ -45,30 +45,29 @@ if [ -n "$MERGES_ARG" ] && [ -n "$SUMMARY_BY_LINE" ]; then
     exit 1
 fi
 
-
+commit="HEAD"
 if [ -n "$SUMMARY_BY_LINE" ]; then
   paths=( "$@" )
 else
-  commit="HEAD"
   [ $# -ne 0 ] && commit=$*
 fi
 project=${PWD##*/}
 
 #
 # get date for the given <commit>
 #
-date() {
+commit_date() {
   # the $1 can be empty
   # shellcheck disable=SC2086
```

Comment:
We can remove the shellcheck now?

Reply:
I removed one extra one, but the rest cannot be removed because of `MERGES_ARG`.

Reply:
It seems we can quote MERGES_ARG as we don't expect any whitespace in it.

Reply:
Unfortunately not because that would mean an empty argument will be passed to `git`. It's funtionally similar to doing this:

```sh
$ git log '' --oneline HEAD
fatal: ambiguous argument '': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
```

Reply:
In other words it shouldn't be quoted so the whitespace can be skipped by the shell during word splitting.

Reply:
I see. Thanks for clarifying that.